.github/workflows/migration-check.yml

```
@@ -1,44 +1,69 @@
+
+      - name: Check migration safety
+        if: steps.filter.outputs.migrations == 'true'
+        id: check
+        timeout-minutes: 10
+        run: |
+          cd backend
+          python manage.py check_migration_safety
+
+      - name: Comment PR
+        if: steps.filter.outputs.migrations == 'true' && (success() || failure())
+        uses: actions/github-script@v6
```

actionlint (1.7.12)

[error] the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)